Project-I-Social-Card/social-card/src/components/FooterComponents/Footer.js

```diff
@@ -1,28 +1,125 @@
 import React from 'react';
 import './Footer.css';
 
+const icons = [
+    {
+        id: 1,
+        name: 'icon-image comment',
+        textClass: 'footer-text-comment',
+        eventFunction: commentClicked,
+        src: 'https://img.icons8.com/ios/30/000000/speech-bubble.png',
+        text: null,
+    },
+    {
+        id: 2,
+        name: 'icon-image refresh',
+        textClass: 'footer-text-refresh',
+        eventFunction: refreshClicked,
+        src: 'https://img.icons8.com/ios-glyphs/30/000000/refresh.png',
+        text: 6,
+    },
+    {
+        id: 3,
+        name: 'icon-image heart',
+        textClass: 'footer-text-heart',
+        eventFunction: heartClicked,
+        src: 'https://img.icons8.com/ios/30/000000/hearts.png',
+        text: 4,
+    },
+    {
+        id: 4,
+        name: 'icon-image envelope',
+        textClass: 'footer-text-envelope',
+        eventFunction: envelopeClicked,
+        src: 'https://img.icons8.com/ios/30/000000/new-post.png',
+        text: null,
+    }
+];
+
+function commentClicked() {
+    const commentIcon = document.querySelector('.comment');
```

Comment:
love the conditional styling, keep this in mind for when you move on to styled components, you're going to love it. It let's you do this so much easier without having to create click events.